CMakeLists.txt

```diff
@@ -91,20 +92,23 @@ if(NOT FFTW3_FOUND OR NOT BLAS_FOUND)
 	message(ERROR "fftw3 or BLAS required to build aubio, but they cannot be found.")
 endif()
 
-if (BLAS_FOUND)
-	if (UNIX AND NOT APPLE)
-		set(AUBIO_CFLAGS "${AUBIO_CFLAGS} -DHAVE_CBLAS_H")
-	endif()
-endif()
-
 include(ExternalProject)
 ExternalProject_Add(aubio
 	SOURCE_DIR ${CMAKE_SOURCE_DIR}/3rdparty/aubio/
-	DOWNLOAD_COMMAND ""
-    UPDATE_COMMAND ""
-	TMP_DIR tmp
-	CONFIGURE_COMMAND "${CMAKE_SOURCE_DIR}/3rdparty/build_aubio.sh" ${ACCELERATE_COMMAND} <INSTALL_DIR> ${AUBIO_CFLAGS}
-	BUILD_COMMAND ""
+        BINARY_DIR ${CMAKE_BINARY_DIR}/3rdparty/aubio-build/
+        INSTALL_DIR ${CMAKE_BINARY_DIR}/3rdparty/aubio-install/
+        UPDATE_COMMAND ""
+	TMP_DIR ${CMAKE_BINARY_DIR}/3rdparty/aubio-build/tmp/
+	CONFIGURE_COMMAND ""
+	BUILD_COMMAND cmake -E env CFLAGS=${AUBIO_CFLAGS} ${CMAKE_SOURCE_DIR}/3rdparty/aubio/waf
```

Comment:
I'd change `cmake` to `${CMAKE_COMMAND}`
As per cmake docs;
> This is the full path to the CMake executable cmake(1) which is useful from custom commands that want to use the cmake -E option for portable system commands. (e.g. /usr/local/bin/cmake)

Reply:
yes you're right